decentraland/components/presentational/Estate.js

```diff
@@ -22,7 +22,7 @@ Estate.propTypes = {
 
 const styles = StyleSheet.create({
   landImage: {
-    width: responsiveWidth(60),
+    width: responsiveWidth(95),
     height: responsiveHeight(20),
```

Comment:
There weren't any height changes required?

Reply:
For me, the way that it now likes good.

Reply:
Okay, sounds good. I'll take a look the next time I run it locally. Resolving this.